src/backend/distributed/commands/create_distributed_table.c

```diff
@@ -86,6 +86,11 @@
  */
 #define LOG_PER_TUPLE_AMOUNT 1000000
 
+#define UNDISTRIBUTE_TABLE_CASCADE_HINT \
+	"Use cascade option to undistribute all the relations involved in " \
+	"a foreign key relationship with \"%s\" by executing SELECT " \
```

Comment:
this doesn't work well with quoted schema/table names, please fix it:
```SQL

ERROR:  cannot undistribute table because it has a foreign key
HINT:  Use cascade option to undistribute all the relations involved in a foreign key relationship with "partitioned_table\'_1" by executing SELECT undistribute_table('partitioned_table\'_1', cascade=>true)
Time: 3.370 ms
```